internal/services/keyvault/key_vault_secret_resource.go

```diff
@@ -88,6 +88,14 @@ func resourceKeyVaultSecret() *pluginsdk.Resource {
 
 			"tags": tags.Schema(),
 		},
+
+		CustomizeDiff: pluginsdk.CustomizeDiffShim(func(_ context.Context, diff *pluginsdk.ResourceDiff, _ interface{}) error {
```

Comment:
is there a reason you didn't just modify the schema?

could we not make a new function and then
```
"tags": tags.SchemaWithMax(15),
```

?

Reply:
Thanks for your feedback. Fixed.